src/test/java/se/diabol/jenkins/pipeline/DeliveryPipelineViewTest.java

```diff
@@ -84,6 +83,7 @@
 
     @Rule
     public JenkinsRule jenkins = new JenkinsRule();
+    private final static String NONE = null;
 
     @Test
     public void testOnJobRenamed() throws Exception {
```

Comment:
lastJob can be renamed, onJobRename in DeliveryPipelineView should handle lastJob renames.